.storybook/preview.tsx

```diff
@@ -1,20 +1,30 @@
-import React from 'react'
+import React, { useCallback } from 'react'
 import I18n from '@scaleway/use-i18n'
 import { Story } from '@storybook/react'
 import { css, ThemeProvider, Global, Theme } from '@emotion/react'
 import { mockRandom } from 'jest-mock-random'
 import normalize from '../src/utils/normalize'
+import { useDarkMode } from 'storybook-dark-mode'
+import { themes } from '@storybook/theming'
+import { light, dark } from './scaleway'
 
-import theme from '../src/theme'
+import lightTheme from '../src/theme'
+import DocsContainer from './components/DocsContainer'
 
 if (process.env.STORYBOOK_ENVIRONMENT === 'visual') mockRandom([0.25, 0.5])
 
 const STORY_SORT = {
   order: ['Home', 'Testing', 'Theme', 'Components'],
 }
 
+const darkMode = {
+  dark: { ...themes.dark, ...dark },
+  light: { ...themes.normal, ...light },
+}
```

Comment:
Strange to have a variable named `darkMode` with both dark and light themes 🤔

Reply:
@QuiiBz Don't ask me thats the variable name of storybook haha